gcodestat.c

```diff
@@ -22,7 +22,14 @@
 #include <string.h>
 #include <stdbool.h>
 #include <math.h>
-#include <malloc.h>
+
+#ifdef __APPLE__ 
+  #include <stdlib.h>
+#else
+  #include <malloc.h>
+#endif
+
+#include <stdlib.h>

```

Comment:
you are including stdlib.h twice in case of __APPLE__

maybe you wanted

#ifndef __APPLE__
  #include <malloc.h>
#endif

weird that there's no malloc on osx?